### ISSUE TITLE
Outdated IRC links

### ISSUE THREAD

**Issue by pdurbin** [opensourcedesign/opensourcedesign.github.io#170]
<img width="892" alt="screen shot 2018-04-19 at 9 23 21 pm" src="https://user-images.githubusercontent.com/21006/39025913-f35ca84e-4417-11e8-8495-d50a8f7c43c0.png">

<img width="892" alt="screen shot 2018-04-19 at 9 23 31 pm" src="https://user-images.githubusercontent.com/21006/39025912-f34e0a28-4417-11e8-9529-c1dbd7275633.png">

**Comment by alexgleason** [comment]
I think #169 resolved this, but maybe it hasn't been deployed

**Comment by pdurbin** [comment]
Interesting. Thanks. My take on #169 is that Open Source Design now prefers Riot/Matrix over IRC but maybe I'm misunderstanding. I still haven't invested time in Riot/Matrix and I'm happy with IRC (I'm a weechat user) so I'm a little sad or at least confused about this change.

I was thinking about how #127 was related in the sense that I already fixed a bad link to https://chat.opensourcedesign.net a while back.

**Comment by alexgleason** [comment]
The Matrix chat is bridged with the IRC chat. I don't think linking to Riot here signifies an overall preference towards Matrix by the community. Rather, Riot is a better user experience for newcomers, so it makes sense to link to the most user-friendly version of the chat.

I think the IRC information should be somewhere on the site, but I wasn't sure where to put it. I think it should be placed in a way that IRC users can find it, but it also doesn't distract newcomers from the Riot room.

I am making the assumption that IRC users are more knowledgeable about tech and therefore need less hand holding to find what they need.

**Comment by pdurbin** [comment]
Currently http://opensourcedesign.net/faq/ ends with "Why are you using IRC":

<img width="586" alt="screen shot 2018-04-19 at 10 50 20 pm" src="https://user-images.githubusercontent.com/21006/39028322-0ffaaf9e-4424-11e8-98c3-a5d4904929c5.png">

This seems like a natural place to update to explain the switch to Riot.

**Comment by RDIL** [comment]
Works for me.

**Comment by RDIL** [comment]
I'll do it actually since I have time to waste

**Comment by RDIL** [comment]
@pdurbin thanks for noticing this, it is fixed now 😁

**Comment by pdurbin** [comment]
@RDIL thanks. After pull request #217 was merged I ran `ack -i irc` and found that IRC is still mentioned on five pages:

governance.md:46:- Do check on IRC if anyone can give you advice or backstory on the issue

CONTRIBUTING.md:18:* Community: Open Source Design is first and foremost a community of people. We hang out in IRC

goals.md:21:  * Easier way to use IRC

code-of-conduct.md:66:This code of conduct especially pertains to our online community platforms (IRC: [#opensourcedesign](http://chat.opensourcedesign.net), 

by-laws.md:48:3. Have you asked on IRC or on GitHub whether it's been discussed before?
